client/ayon_core/hosts/substancepainter/plugins/create/create_textures.py

```diff
@@ -88,8 +98,25 @@ def create_instance_in_context_from_existing(self, data):
         return instance
 
     def get_instance_attr_defs(self):
-
+        layer_stack_channel_enum = ["BaseColor", "Metallic", "Roughness",
+                                    "Normal", "Height", "Specular",
+                                    "SpecularEdgeColor", "Emissive", "Opacity",
+                                    "Displacement", "Glossiness", "Anisotropylevel",
+                                    "AO", "Anisotropyangle", "Transmissive",
+                                    "Reflection", "Diffuse", "Ior",
+                                    "Specularlevel", "BlendingMask", "Translucency",
+                                    "Scattering", "ScatterColor", "SheenOpacity",
+                                    "SheenRoughness", "SheenColor", "CoatOpacity",
+                                    "CoatColor", "CoatRoughness", "CoatSpecularLevel",
+                                    "CoatNormal"]
         return [
+            EnumDef("exportChannel",
+                    items=layer_stack_channel_enum,
+                    multiselection=True,
+                    default=None,
+                    label="Export Channel(s)",
+                    tooltip="Choose the channel which you "
+                            "want to solely export"),
```

Comment:
Might want to add to the tooltip thta if NONE were selected that ALL available channels in a preset are exported?